tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     /* Basic Options */
-    "target": "es6",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "target": "es2018",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
```

Comment:
Should lib be updated in synch with target?  Current recommendations for node 12 are [in this repo](https://github.com/tsconfig/bases/tree/master/bases).  fyi, have been running with latest node without issues so we may want to consider updating the node version in package.json.

Reply:
> 
> 
> Should lib be updated in synch with target? Current recommendations for node 12 are [in this repo](https://github.com/tsconfig/bases/tree/master/bases). fyi, have been running with latest node without issues so we may want to consider updating the node version in package.json.

Makes sense! Pushed.